index.html

```diff
@@ -1,10 +1,44 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>Lesson 8</title>
+        <link rel="icon" type="image/x-icon" href="https://cdn-icons-png.flaticon.com/512/119/119598.png">
+        <title>Lesson 7 - version 7.1</title>
         <link href="style.css" rel="stylesheet" type="text/css">
     </head>
     <body>
-        <h1>Lesson 8</h1>
+        <h1>Veronica Rubliuk</h1>
+        <img src="img.jpg" width="210" height="260" alt="lesson_7_robot.dreams_rub_ver">
+        <h3>junior front-end developer</h3>
+        <div>Work Expirience</div>
+        <br>
+        <table>
+            <thead>
+                <tr class="thHead">
+                    <th colspan="1">Period</th>
+                    <th colspan="3">Position</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr style="height:50px" class="tdBody">
```

Comment:
write styles inside of html is very bad practice, please move it to .css file

Reply:
done